linode_api4/objects/base.py

```diff
@@ -99,6 +100,18 @@ def _expand_vals(self, target, **vals):
     def __repr__(self):
         return "Mapping containing {}".format(vars(self).keys())
 
+    @staticmethod
+    def _flatten_base_subclass(obj: "Base") -> Optional[Dict[str, Any]]:
+        if obj is None:
+            return None
+
+        # If the object hasn't already been lazy-loaded,
```

Comment:
Just curious, in which cases that a object isn't lazy-loaded?

Reply:
The most notable example would be the underlying Disk/Volume objects for an Instance config's `device` mapping ^